data.sql

```diff
@@ -0,0 +1,7 @@
+/* Populate database with sample data. */
+
+insert into animals (animal_id,animal_name,date_of_birth,weight_kg,neutered,escape_attempts)
+values (102,'Agumon','2020/02/03',10.23,true,0),
+       (102,'Gabumon','2018/11/15',8,true,2),
```

Comment:
- [ ] Two rows in your table can't have the same id `animal_id`, I'll suggest you correct this or make the changes to the `schema` file suggested above and the id column will be generated automatically. :+1: :100: